src/diffusers/models/transformers/transformer_flux.py

```diff
@@ -83,11 +83,12 @@ def forward(
         hidden_states: torch.FloatTensor,
         temb: torch.FloatTensor,
         image_rotary_emb=None,
+        joint_attention_kwargs=None,
     ):
         residual = hidden_states
         norm_hidden_states, gate = self.norm(hidden_states, emb=temb)
         mlp_hidden_states = self.act_mlp(self.proj_mlp(norm_hidden_states))
-
+        joint_attention_kwargs = joint_attention_kwargs if joint_attention_kwargs is not None else {}
```

Comment:
should we pass this to `attn` too?

Reply:
Yes! I think it will be useful for other trial!